perun/cli.py

```diff
@@ -105,6 +104,9 @@
     is_flag=True,
     help="Disables the colored output.",
 )
+@click.option(
+    "--say-yes", "-y", default=False, is_flag=True, help="Says yes to every confirmation prompt"
+)
```

Comment:
Just a small detail, there seems to be a missing dot at the end (all the other help strings have it).

Reply:
Good catch, however, I will not fix it, as I have a plan for rewritting/reworking the help messages (and maybe CLI commands as well). I want to unify it mostly. Btw. I have already asked AI to rewrite some of the messages, however, I was not really happy with it, so I will rewrite it myself manually, probably during friday/next weekend.